stratum/hal/lib/common/common.proto

```diff
@@ -1034,7 +1024,7 @@ message FrontPanelPortInfo {
   HwState hw_state = 6;
 }
 
-// Optical channel configuration
+// Optical channel state and configuration.
 message OpticalChannelInfo {
```

Comment:
I'm still not sure on the `OpticalChannelInfo` name. The channel part only seems to be a small subset of the parameters.

`OpticalNetworkInterfaceInfo` is a bit too long. Maybe `OpticalTransceiverInfo`?

Reply:
Agree, OpticalChannelInfo may be misleading. We can use OpticalTransceiverInfo since we are describe the current state of an optical transceiver.